syntaxes/idris.tmLanguage

```diff
@@ -121,6 +121,12 @@
 			</array>
 		</dict>
 		<dict>
+			<key>match</key>
+			<string>(?&lt;!\w)\'\w++(?!')</string>
```

Comment:
Idris have introduced tagged effects using this syntax:
```idris
whatever : Eff ['a ::: STATE, 'b ::: STATE] ()
whatever = do
    x <- ('a :- get)
    'b :- (put x)
```
So we have to support `'identifier` syntax.